test/Sentry.Tests/Protocol/MeasurementTests.verify.cs

```diff
@@ -0,0 +1,29 @@
+#if !__MOBILE__
```

Comment:
How about excluding *verify.cs* files in the directory build props for mobile projects/targets? Would avoid the ifdef in all files